app/controllers/spree/admin/gdpr_requests_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Spree
+  module Admin
+    class GdprRequestsController < ResourceController
+      before_action :load_data, only: :new
+
+      def index
+        respond_with(@collection, &:html)
+      end
+
+      def run
+        @object&.serve
```

Comment:
Why the `&.`? If `@object` is not defined, we should probably handle that case and show an error.